app/templates/base.html

```diff
@@ -63,6 +63,26 @@
                     }
                 )
             });
+        function set_message_count(n) {
+            $('#message_count').text(n);
+            $('#message_count').css('visibility', n ? 'visible' : 'hidden');
+        }
+        {% if current_user.is_authenticated %}
+        $(function() {
+            var since = 0;
```

Comment:
please avoid using `var` keyword when you declare variables because this is considered bad practice. Better use let (for variable) or const (for constant)
Check similar `var` use cases in this file